backend/src/main/kotlin/hu/bme/sch/cmsch/component/pushnotification/PushNotificationService.kt

```
@@ -79,32 +87,90 @@ class PushNotificationService(
+            return
+        }
+
+        try {
+            log.debug("Inserting messaging token for user: {}, token: {}", userId, token)
+            messagingTokenRepository.save(MessagingTokenEntity(
+                userId = userId,
+                token = token,
+                createdAt = now,
+                updatedAt = now
+            ))
+        } catch (e: DataIntegrityViolationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)